receiver/signalfxreceiver/receiver.go

```diff
@@ -46,14 +46,16 @@ import (
 const (
 	defaultServerTimeout = 20 * time.Second
 
-	responseOK                 = "OK"
-	responseInvalidMethod      = "Only \"POST\" method is supported"
-	responseInvalidContentType = "\"Content-Type\" must be \"application/x-protobuf\""
-	responseInvalidEncoding    = "\"Content-Encoding\" must be \"gzip\" or empty"
-	responseErrGzipReader      = "Error on gzip body"
-	responseErrReadBody        = "Failed to read message body"
-	responseErrUnmarshalBody   = "Failed to unmarshal message body"
-	responseErrNextConsumer    = "Internal Server Error"
+	responseOK                      = "OK"
+	responseInvalidMethod           = "Only \"POST\" method is supported"
+	responseInvalidContentType      = "\"Content-Type\" must be \"application/x-protobuf\""
+	responseInvalidEncoding         = "\"Content-Encoding\" must be \"gzip\" or empty"
```

Comment:
Should you create variables instead of calling initJSONResponse every time?

Reply:
The only diff here is the last two lines, they just triggered a reformatting that added more spaces to these lines.